plugins/lua/orders.lua

```diff
@@ -280,7 +280,7 @@ local function can_set_skill_level()
             return false
         end
     end
-    return ret
+    return ret and not df.global.game.main_interface.stockpile_link.adding_new_link
```

Comment:
I think this should instead be fixed in the focus string, since the focus string is not accurately representing the UI state. I'll see if I can do that.

Reply:
fixed in https://github.com/DFHack/dfhack/pull/4858 -- you should be able to remove this check now